src/pages/billing/Project.js

```diff
@@ -308,7 +319,56 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
               isClearable: false,
               options: _.map(({ displayName, accountName }) => ({ label: displayName, value: accountName }), billingAccounts),
               onChange: ({ value: newAccountName }) => setSelectedBilling(newAccountName)
+            }),
+            div({ style: { marginTop: '1rem' } }, ['Note: Changing the billing account for this ' +
+            'billing project will clear the spend report configuration.'])
+          ])])
+        ])
+      ]),
+      div({ style: { color: colors.dark(), fontSize: 14, display: 'flex', alignItems: 'center', marginTop: '0.5rem', marginLeft: '1rem' } }, [
+        span({ style: { flexShrink: 0, fontWeight: 600, fontSize: 14, margin: '0 0.75rem 0 0' } }, 'Spend Report Configuration:'),
```

Comment:
And when it really is only one, perhaps:
```suggestion
        span({ style: { flexShrink: 0, fontWeight: 600, fontSize: 14, marginRight: '0.75rem' } }, 'Spend Report Configuration:'),
```